infra/cmake/packages/CaffeConfig.cmake

```diff
@@ -20,7 +20,7 @@ function(_Caffe_import)
     return()
   endif()
 
-  find_package(HDF5 COMPONENTS HL QUIET)
+  nnas_find_package(HDF5 QUIET)
```

Comment:
Others used `COMPONENTS CXX ` and Caffe uses `COMPONENTS HL` but this information is gone.
Is this OK? Have you checked what this does?

Reply:
Maybe that keyword divides libraries as what user want to. Since I build all of them, that keyword becomes unnecessary.
```cmake
set(HDF5_LIBRARIES "hdf5${_SUFFIX}")
set(HDF5_C_LIBRARIES "hdf5${_SUFFIX}")
set(HDF5_CXX_LIBRARIES "hdf5_cpp${_SUFFIX}")
set(HDF5_HL_LIBRARIES "hdf5_hl${_SUFFIX}")
set(HDF5_C_HL_LIBRARIES "hdf5_hl${_SUFFIX}")
set(HDF5_CXX_HL_LIBRARIES "hdf5_hl_cpp${_SUFFIX}")
```